lib/matplotlib/backend_bases.py

```diff
@@ -2791,6 +2791,12 @@ def mouse_move(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                ax = event.inaxes
+                artists = ax.artists + ax.images + ax.lines
```

Comment:
Ok then. So how to handle composite images?  It does not seem feasible to recreate the composite every time.